README.md

```diff
@@ -233,4 +233,6 @@ This table lists the modules recipy has patches for, and the input and output fu
 </tbody>
 </table>
 
+
```

Comment:
Could I be a right pain and ask you to remove these two blank lines that I added as an example earlier?